dependencies/influxdb/service.go

```diff
@@ -0,0 +1,154 @@
+// TODO: copied from "github.com/influxdata/influxdb-client-go/v2/api/http", contribute back changes instead
```

Comment:
Can you either put this file into an internal package within this directory or avoid exporting anything in this file? The packages in this directory are particularly susceptible to API changes because they are intended to be implemented by third-parties that are embedding flux. I'd like to avoid creating a contract for a fork of a dependency.

Reply:
I plan to remove this file before merge - the only difference from the copy is that it takes a `Doer` interface instead of a `http.Client` struct. I think this can be changed upstream before merge.

Reply:
Specifically I need this PR merged upstream: https://github.com/influxdata/influxdb-client-go/pull/256/files

Reply:
Actually let's not block this on the other PR - I put in a TODO referencing the upstream PR and moved the service implementation to `internal`